### PR TITLE
Object detection

### PR DESCRIPTION
**Pull request by msabdo**
thanks for god and for you,  you can use init and detect functions  to deals with model

### CODE REVIEW

File: objectDetection_lite.py [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+              'مزهرية', 'مقص', 'دمية دب', 'مجفف شعر', 'فرشاة الأسنان']
+
+    # Load TFLite model and allocate tensors.
+    interpreter = tf.lite.Interpreter(model_path="model2.tflite")
+    interpreter.allocate_tensors()
+
+    return interpreter, labels
+
+
+def detect(image, interpreter, labels):
```
Comment (mhashim6) [inline objectDetection_lite.py]: again, the caller shouldn't have the interpreter in the first place, your module should have it stored. this function should only take the image, and the labels.

File: objectDetection_lite.py [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+              'بيتزا', 'الدونات', 'كيك', 'كرسي', 'كنبة', 'النبات المحفوظ بوعاء', 'السرير', 'طاولة الطعام', 'الحمام',
+              'رصد التلفزيون', 'حاسوب محمول', 'الفأر', 'التحكم عن بعد', 'لوحة المفاتيح',
+              'الهاتف الخلوي', 'الميكروويف', 'فرن', 'محمصة خبز كهربائية', 'مكتب المدير', 'ثلاجة', 'كتاب', 'ساعة حائط',
+              'مزهرية', 'مقص', 'دمية دب', 'مجفف شعر', 'فرشاة الأسنان']
+
+    # Load TFLite model and allocate tensors.
+    interpreter = tf.lite.Interpreter(model_path="model2.tflite")
+    interpreter.allocate_tensors()
+
+    return interpreter, labels
```
[the remark this answers is not in the document]
Comment (msabdo) [reply objectDetection_lite.py]: to load the model once and and use the interpreter next in interpreter.invok function

[the remark this answers is not in the document]
Comment (mhashim6) [reply objectDetection_lite.py]: Yeah I know, that's not the point. You shouldn't return the interpreter. instead, maybe store it in a global variable.

File: objectDetection_lite.py (hunk 1 above)
Comment (msabdo) [reply objectDetection_lite.py]: the problem is that the function that load model is the same function that return the reference to the object that use for detection

Comment (mhashim6) [reply objectDetection_lite.py]: I understand. Maybe store it globally and then access it in this function. Maybe give it an initial value of `None`.

**Comment by mhashim6** [comment]
> thanks for god and for you, you can use init and detect functions to deals with model

Oh! You're welcome!

File: objectDetection_lite.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (msabdo) [reply objectDetection_lite.py]: ok . it's done

[the remark this answers is not in the document]
Comment (mhashim6) [reply objectDetection_lite.py]: Nice! now push the changes.

**Comment by mhashim6** [comment]
Really nice work, merged to `devel`. You can delete your local branch now.

**Pull request merged**